package.json

```diff
@@ -28,5 +28,16 @@
     "cross-env": "^3.2.3",
     "lerna": "^2.0.0-beta.38-31-g0749ea6",
     "nyc": "^10.2.0-candidate.0"
+  },
+  "nyc": {
+    "exclude": [
+      "**/test/**",
+      "**/dist/**",
+      "**/bin/**"
+    ],
+    "reporter": [
+      "text",
+      "lcov"
```

Comment:
Added this for a local verification. I'm not overly attached to it, but `text` and `lcov` are usually the reporters I have on every project... but I am open to removing it for the sake of performance or some other valid reason.